onnxmltools/convert/common/_apply_operation.py

```diff
@@ -0,0 +1,312 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# This file contains some high-level APIs for applying operations on variables specified by names. We should try our
+# best to use those functions because they can produce ONNX operators according to the ONNX version specified in the
+# `container` argument. Notice that those function behaviors are defined in a way very similar to ONNX-1.2.
+
+from distutils.version import StrictVersion
+from ...proto import onnx_proto
+
+
+def _create_name_or_use_existing_one(scope, op_type, name):
+    if name is None:
+        return scope.get_unique_operator_name(op_type)
+    else:
+        return name
```

Comment:
>return name [](start = 8, length = 11)

Does this need to be checked for C-style naming?

Reply:
This function only consumes strings produced by our `get_unique_operator_name` and `None,` so I didn't check the input string.